test/hotspot/jtreg/runtime/jni/nativeStack/TestNativeStack.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please change the copyright to read "2022, 2023," - thanks.

Reply:
Updated, thank you!!!